server/src/services/generatorFlow.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable consistent-return */
+import { generator } from './generator';
+import { channelMessageValidator, isJson } from '../utils/schemaValidators';
+import schemaCreator from '../utils/schemaCreator';
+import defaultSpec from '../models/defaultSpec.json';
+import questions from '../models/questions';
+import { childEntityValue } from '../utils/entities';
+
+const document = defaultSpec;
+
+export default function generatorFlow(
```

Comment:
Could you split this big function to the separate ones? I mean, to step-functions?

Reply:
This function is very big and it's hard to read and understand, so please move some logic to separate functions - step-functions.

Reply:
Ok, it's a POC so @AceTheCreator, you can leave as it is and fix it in the next PRs.